pkg/proxy/proxy.go

```diff
@@ -94,15 +79,47 @@ func (p *Proxy) Create(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Pushing LoadTest to Kubernetes
-	loadTestName, err := p.kubeClient.CreateLoadTest(ctx, loadTest)
-	if err != nil {
-		if errors.Is(err, os.ErrExist) {
+	// Find the old load test with same data
+	labeledLoadTests, err := p.kubeClient.GetLoadTestsByLabel(ctx, loadTest)
```

Comment:
You are not checking errors here, is that right?

Reply:
Just changed the lines but there is error checks for this line also below

Reply:
The error is being overwritten, so not tested